.github/workflows/amazon_bedrock.yml

```diff
@@ -27,7 +27,6 @@ env:
   PYTHONUNBUFFERED: "1"
   FORCE_COLOR: "1"
   AWS_REGION: us-east-1
-  HF_TOKEN: ${{ secrets.HF_TOKEN }}
```

Comment:
this is no longer needed. In the past our integration depended on HF for tokenization and we tested some HF models.